#ifndef USER_PARAMS_H
#define USER_PARAMS_H

#include <eigen3/Eigen/Eigen>

namespace task_01_controller
{

typedef struct
{

  double param1;
  double param2;
  double param3;
  double param4;
  double param5;
  double param6;
  double param7;
  double param8;
  double param9;
  double param10;
  double param11;
  double param12;

} UserParams_t;

}  // namespace task_01_controller

#endif  // USER_PARAMS_H
